### DISCUSSION TITLE
Sida för försäljning

### DISCUSSION THREAD

**Discussion by CL-Webbunderhallare** [Webbutvecklare-CL/cl-sektionen.se#144] [Ideas]
Försäljningsansvarig kan ha någon typ av webbshop.

Försäljningsansvarig kan lägga upp olika produkter med bild och pris i olika kategorier. Kanske kan skapa kategorier. Detta görs i personalrummet. Detta laddas upp i firebase.

Produkterna hämtas på samma sätt som aktuelltinläggen. På sidan kanske det kan finnas en länk till ett google formulär där man kan ladda upp bilder på märkesförslag eller liknande.

**Comment by Moppe-xd** [comment]
Kan inflika att det skulle bara vara nice att ha en sida där jag visa vad som finns till försäljning och hur mycket allting kostar. Ser detta som typ i MVP
